ryhope/src/storage/pgsql/mod.rs

```diff
@@ -490,7 +586,45 @@ where
             )
             .await
             .map(|_| ())
-            .with_context(|| format!("unable to create table `{table}_meta`"))
+            .with_context(|| format!("unable to create table `{table}_meta`"))?;
+
+        // Create the mapper table if the mapper table is not external, otherwise
+        // create a view for the mapper table name expected for `table` to `mapper_table`.
+        if EXTERNAL_EPOCH_MAPPER {
+            ensure!(
+                mapper_table.is_some(),
+                "No mapper table name provided for storage with external epoch mapper"
+            );
+            let mapper_table_alias = mapper_table_name(table);
+            let mapper_table_name = mapper_table_name(mapper_table.unwrap().as_str());
+            connection
+                .execute(
+                    &format!(
+                        "
+                        CREATE VIEW {mapper_table_alias} AS
+                        SELECT * FROM {mapper_table_name}"
```

Comment:
I don't think this should be `*`?

Reply:
Why not? This should be a view of the entire table. Or you mean you would like to see explicit column names as in the comment above?

Reply:
Yeah, sorry, I meant I would prefer an explicit list of columns.

Reply:
Done in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/425/commits/9a606e9c9e8b321fc79145d50371e345d336870a